### PR TITLE
M365 Dashboard card styling changes

### PR DESCRIPTION
**Pull request by veerashayyagari**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

1. Fix body text styles and card frame overflow
2. Include lastupdatedon text for dataviz component
3. Update recommendation card component based on latest card frame
4. Add Recommendation card examples with MultiStackedBarChart.

### CODE REVIEW

File: packages/dashboard/src/components/Card/Layout/Layout.tsx [hunk 1]
```diff
@@ -102,20 +106,24 @@ export class Layout extends React.Component<ILayoutProps> {
...
-                    legendColors={legendColors}
-                    barWidth={barWidth}
-                    data={data}
-                    dataPoints={dataPoints}
-                    compactChartWidth={compactChartWidth}
-                    width={this._getChartWidth(cardContentList.length)}
-                    height={this._getChartHeight(cardContentList.length)}
-                  />
+                  <React.Fragment>
+                    {chartUpdatedOn && <div className={dataVizLastUpdateClassName}>{chartUpdatedOn}</div>}
```
Comment (KevinTCoughlin) [inline packages/dashboard/src/components/Card/Layout/Layout.tsx]: Just a non-blocking suggestion to consider [\<time>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time) with the `datetime` attribute and optionally `title` for hover (Reddit post timestamps do this).

File: packages/dashboard/src/components/Card/examples/Card.Chart.MultiStackedBar.Example.tsx [hunk 2]
```diff
@@ -46,13 +46,15 @@ export class MultiStackedBarChartExample extends React.Component<{}, {}> {
...
+      chartUpdatedOn: 'Updated 6:20 pm today',
       compactChartWidth: 394,
       legendColors: colors
     };
 
     const textContent = {
       subHeaderText: 'Edit data loss prevention policies',
-      bodyText: 'Information based on your data loss prevention (DLP) policies'
+      bodyText:
+        'Information based on your data loss prevention (DLP) policies may take upto 24 hours to appear in this activity report'
```
Comment (KevinTCoughlin) [inline packages/dashboard/src/components/Card/examples/Card.Chart.MultiStackedBar.Example.tsx]: Super nit but "up to" is grammatically correct vs. "upto"

Comment (veerashayyagari) [reply packages/dashboard/src/components/Card/examples/Card.Chart.MultiStackedBar.Example.tsx]: Updated :)

File: packages/dashboard/src/components/Card/Layout/Layout.tsx (hunk 1 above)
Comment (veerashayyagari) [reply packages/dashboard/src/components/Card/Layout/Layout.tsx]: Currently that design team wanted this to be a free text to give the flexibility to show something like 'last updated today' or 'last updated yesterday'.

**Pull request merged**